packages/lib/factorio/server.js

```diff
@@ -517,12 +538,27 @@ class FactorioServer extends events.EventEmitter {
 		this.rconPassword = options.rconPassword;
 		/** Enable player whitelist */
 		this.enableWhitelist = options.enableWhitelist;
+		/** Enable verbose logging */
+		this.verboseLogging = options.verboseLogging;
 		this._state = "new";
```

Comment:
For fixed things like `_state` here probably also a good idea to make it actual constants and use those.
In PHP I would make it part of the class that's responsible for it, idk how best to do it in JS though.